recipes/mpld3/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2" %}
+
+package:
+    name: mpld3
+    version: {{ version }}
+
+source:
+    fn: mpld3-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/m/mpld3/mpld3-{{ version }}.tar.gz
+    md5: 266f4e51a094f8ff1aaf7e2018f763b4
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - mplexporter
+    run:
+        - python
+        - jinja2
+        - matplotlib
+        - mplexporter
+
+test:
+    imports:
+        - mpld3
+
+about:
+    home: http://mpld3.github.com
+    license: BSD-3-Clause
+    summary: D3 Viewer for Matplotlib
+
+extra:
+    recipe-maintainers:
+        - jakevdp
```

Comment:
@jakevdp is it OK if I add you as a maintainer here? Are there more people you would like to add?